ui/src/app/pages/data/concept/concept-set-details.tsx

```diff
@@ -138,7 +138,8 @@ export const ConceptSetDetails = fp.flow(withUrlParams(), withCurrentWorkspace()
     }
 
     componentDidMount() {
-      this.getConceptSet();
+      workspacesApi().updateRecentWorkspaces(this.props.workspace.namespace, this.props.workspace.id)
```

Comment:
Did you consider adding this logic centrally instead, e.g. here? https://github.com/all-of-us/workbench/blob/900231bdd366a3143d3c51f030f380521055e8f4/ui/src/app/pages/workspace/workspace-wrapper/component.ts#L23

Any workspace-related component is supposed to be rendered via that codepath. It's in Angular, but we'll likely have a logically corresponding React component once we complete the migration. Alternatively, as part of the withCurrentWorkspace() HOC.